backend/app/tests/services/scheduled_inquiries/test_create_scheduled_inquiry.py

```diff
@@ -38,15 +50,15 @@ def test_create_when_no_scheduled_inquiries_exist_should_assign_rank_1(
 
 
 def test_create_when_previous_highest_rank_is_1_should_assign_rank_2(
-    db: Session, db_inquiry: Inquiry
+    db: Session, db_inquiry_1: Inquiry, db_inquiry_2: Inquiry
 ) -> None:
     # First scheduled inquiry will have rank 1
-    scheduled_inquiries_service.create(session=db, inquiry_id=db_inquiry.id)  # type: ignore
+    scheduled_inquiries_service.create(session=db, inquiry_id=db_inquiry_1.id)  # type: ignore
 
     # Additional scheduled inquiries will be assinged the next highest rank
     next_scheduled_inquiry = scheduled_inquiries_service.create(
         session=db,
-        inquiry_id=db_inquiry.id,  # type: ignore
+        inquiry_id=db_inquiry_2.id,  # type: ignore
```

Comment:
seems like not all functions are tested. For example, move. Is the plan to add those in a follow-up PR?

Reply:
I would like to add them in my next, follow up PR. I want to get these changes in front of Jillian for feedback and available to Rashmi and Milad for QA testing.